.github/workflows/uninstall_install_sandbox.yaml

```
@@ -58,9 +71,22 @@ jobs:
           namespace: ${{ secrets.OPENSHIFT_NAMESPACE }}
 
       - name: Install Traction via Helm
-        continue-on-error: true
+        continue-on-error: false
         run: |
           helm repo add traction https://bcgov.github.io/traction 
           helm upgrade --install traction-sandbox traction/traction \
           -f ./helm-values/traction/values-sandbox.yaml \
-          -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=7m
\ No newline at end of file
+          -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=7m
+
+      - name: Rocket.Chat Notification
+        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@master
```

```suggestion
        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@1.1.1
```
It's best practice to always target a released version of an action, that way you are not surprised by unexpected changes as you would be when targeting the `master` branch.  We covered this during your previous work with this action here; https://github.com/bcgov/von-bc-registries-audit/blob/main/.github/workflows/actions/deploy/action.yml#L83-L93